src/app.py

```diff
@@ -40,12 +41,16 @@ def find():
         language_id = request.form['sellanguage']
         sel_word = request.form['selword']
         language_name = language_dict[language_id]
-        if not appService.udt_pre_model:
-            appService.config_udpipe(language_name)
-        appService.find_service(language_name, sel_word)
+        app_context.sel_word = sel_word
+        app_context.sel_language = language_name
+        if not app_service.udt_pre_model:
+            app_service.config_udpipe(language_name)
+        app_service.find_service(language_name, sel_word)
+        sel_result_kwic = app_service.kwic(sel_word, app_service.sel_result)
+        app_context.sel_result_kwic = sel_result_kwic
     return render_template('result.html', input_data={"language_name": language_name,
                                                       "sel_word": sel_word,
-                                                      "sel_result": appService.sel_result})
+                                                      "sel_result": sel_result_kwic})
 
 
 @app.route('/find2', methods=['POST'])
```

Comment:
This code inside this function is the same as the function above. How about if we can make it generic and make it work using a single function.